pulumi/folio/deployments/json2yaml.ts

```diff
@@ -0,0 +1,14 @@
+import * as fs from 'fs';
```

Comment:
Do you need both `json2yaml.js` and `json2yaml.ts`?

Reply:
The js file is the compiled version of the ts file. I think it is fine that both get committed. I put some instructions for how to compile and use the script in the new readme in this dir.